data/net/minecraft/world/inventory/RecipeBookMenu.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/world/inventory/RecipeBookMenu
 	METHOD fillCraftSlotsStackedContents (Lnet/minecraft/world/entity/player/StackedContents;)V
 		ARG 1 itemHelper
+	METHOD handlePlacement (ZLnet/minecraft/world/item/crafting/Recipe;Lnet/minecraft/server/level/ServerPlayer;)V
+		ARG 1 shouldPlaceAll
```

Comment:
While this parameter is `shouldPlaceAll`, the parameter for `ServerPlaceRecipe#recipeClicked` is named `placeAll`. Should this parameter be changed to follow, or the other parameter be changed?